clang/lib/DPCT/APINamesThrust.inc

```diff
@@ -1416,3 +1416,47 @@ CONDITIONAL_FACTORY_ENTRY(
 CALL_FACTORY_ENTRY("thrust::make_zip_iterator",  CALL("oneapi::dpl::make_zip_iterator",  ARG(0)))
 // thrust::make_reverse_iterator
 CALL_FACTORY_ENTRY("thrust::make_reverse_iterator",  CALL("oneapi::dpl::make_reverse_iterator",  ARG(0)))
+
+// thrust::exclusive_scan
+CONDITIONAL_FACTORY_ENTRY(
+  CheckArgCount(3),
+  // thrust::exclusive_scan(data, data + 6, data)
```

Comment:
Need add additional condition to check the iterator is host/device (like line 1384)